modules/admin_manual/nav.adoc

```diff
@@ -112,21 +114,23 @@
 **** xref:maintenance/encryption/migrating-from-user-key-to-master-key.adoc[Migrating from User Key to Master Key Encryption]
 *** xref:maintenance/migrating.adoc[Migrating to a Different Server]
 *** xref:maintenance/restore.adoc[Restore]
-
 ** xref:appliance/index.adoc[Appliance]
-*** UCS
-**** xref:appliance/ucs/add-groups-and-users.adoc[Add Groups and Users]
-*** xref:appliance/enterprise_trial.adoc[Enterprise Trial]
-*** xref:appliance/Active_Directory.adoc[Active Directory]
-*** xref:appliance/Backup.adoc[Backup]
-*** xref:appliance/Clamav.adoc[ClamAV]
-*** xref:appliance/Office.adoc[Office]
-*** xref:appliance/firewall.adoc[Firewall]
-*** xref:appliance/certificates.adoc[Certificates]
-*** xref:appliance/howto-update-owncloud.adoc[How to Update ownCloud]
-*** xref:appliance/installation.adoc[Installation]
-*** xref:appliance/login_information.adoc[Login Information]
-*** xref:appliance/wnd_setup.adoc[Windows Network Drive Setup]
+*** Installation
+**** xref:appliance/installation/installation.adoc[Installation]
```

Comment:
From a user-journey perspective, it would make more sense to link the header above, instead of having two links with the same name.

Reply:
Hm, I don't quite get it

Reply:
What I meant is instead of lines 118 & 119 above, that it would be as follows:

```asciidoc
*** xref:appliance/installation/installation.adoc[Installation]
**** xref:appliance/installation/active_directory.adoc[Active Directory]
**** ... other appliance links
```

Reply:
@voroyam, I implemented the change in ff126a0. Are you okay with it?

Reply:
sure